dm-manager/pkg/api/dm-manager/dm-manager.proto

```diff
@@ -0,0 +1,57 @@
+// SPDX-FileCopyrightText: (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package device_management;
+
+option go_package = "dm-manager/pkg/api/dm-manager;dm_manager";
+
+// Enum for operation type
+enum OperationType {
+    ACTIVATE = 0;   
+    DEACTIVATE = 1;
+}
+
+// Enum for AMT status
+enum AMTStatus {
+    ENABLED = 0;
+    DISABLED = 1;
+}
+
+// Enum for activation result status.
+enum ActivationStatus {
+    PROVISIONED = 0;
+    FAILED = 1;
+}
+
+// Service for Device Management.
+service DeviceManagement {
+    // PMA reports AMT status to dm manager.
+    rpc ReportAMTStatus(AMTStatusRequest) returns (AMTStatusResponse);
+
+    // DM manager sends activation request to PMA.
+    rpc SendActivationRequest(ActivationRequest) returns (ActivationResponse);
+}
```

Comment:
The 3rd API is still needed to allow the agent to report the status after performing a device activation or deactivation. Request for that API should contain the host ID and a string containing the device activation/deactivation result. Response can be blank